ta/pkcs11/src/pkcs11_attributes.h

```diff
@@ -198,4 +198,7 @@ enum pkcs11_rc set_key_data(struct obj_attrs **head, void *data,
 enum pkcs11_rc add_missing_attribute_id(struct obj_attrs **pub_head,
 					struct obj_attrs **priv_head);
 
+enum pkcs11_rc get_key_data_to_wrap(struct pkcs11_object *obj, void **data,
```

Comment:
There is set_key_data method... so why not  move this next to it?

Reply:
Ok